docs/index.md

```diff
@@ -41,15 +41,15 @@ Details of the Secure Device Onboard protocol can be found in the Secure Device
 
 The Secure Device Onboard project provides software that helps simplify the implementation of an end-to-end Secure Device Onboard capability.  The following components are available:
 
-**Client SDK** - The Client SDK provides devices with an implementation of the DI protocol, which initializes the device with credentials, the TO1 protocol, which connects the device to a Rendezvous Service, and the TO2 Protocol, which connects the device to an IOT Platform Service.  The Client SDK supports Linux on X86 processors, as well as Linus and SELinux on ARM processors.
+**Client SDK** - The Client SDK provides devices with an implementation of the DI protocol, which initializes the device with credentials, the TO1 protocol, which connects the device to a Rendezvous Service, and the TO2 Protocol, which connects the device to an IOT Platform Service.  The Client SDK supports Linux on X86 processors, as well as Linux and SELinux on ARM processors.
 
 **Manufacturing Toolkit** - The Manufacturing Toolkit software enables the manufacturer to load credential into devices.  It implements the DI protocol, and includes support for database storage of credentials and is designed to integrate into the manufacturers IT processes.
 
 **Reseller Toolkit** - The Reseller Toolkit enables distributors and System Integrators to manage the ownership credential over the devices lifetime.  It implements the DI protocol.
 
-**Rendezvous Service** - The Rendezvous Service software provides a complete Rendezvous service, and implements the TO0 and TO1 protocols required for device registration and discovery.  It can be run in the cloud or on-premise and includes an interface to support key attestation.
+**Rendezvous Service** - The Rendezvous Service software provides a complete Rendezvous service and implements the TO0 and TO1 protocols required for device registration and discovery.  It can be run in the cloud or on-premise and includes an interface to support key attestation.
 
-**IOT Platform SDK** - The IOT Platform SDK provides micro-services to facilitate integration of an IOT Platform in the Secure Device Onboard processes. It implements the TOO protocol required for device registration, and the TO2 protocol required for device ownership transfer.
```

Comment:
the noun phrase "Integration" seems to be missing a determiner before it. So adding an article.